applications/ecallmgr/src/ecallmgr.hrl

```diff
@@ -450,6 +450,7 @@
                    ,{'bridge', ['CHANNEL_BRIDGE', 'CHANNEL_UNBRIDGE']}
                    ,{'media', ['DETECTED_TONE', 'DTMF','CHANNEL_PROGRESS','CHANNEL_PROGRESS_MEDIA']}
                    ,{'record', ['RECORD_START', 'RECORD_STOP']}
+                   ,{'playback', ['PLAYBACK_STOP']}
```

Comment:
you added playback start/stop events and only stop here ?

Reply:
We only actually use PLAYBACK_STOP. It is used when an agent receives a callback call. When an agent answers a callback call, an announcement is first played and then on PLAYBACK_STOP event the member is called back. 
We could remove PLAYBACK_START but thought it makes sense to treat as a pair.

Reply:
@lazedo Does it make sense to include PLAYBACK_START as well in this context?  We are happy to remove it.  Please let us know your preference.

Reply:
this doesn't make sense to me and its going to increase the event stream without the need for it, you already have channel_execute_complete fired when a command completes. not willing to accept this change as-is because it will impact everybody with the increased streaming.

Reply:
@lazedo no problem.  We will look into `channel_execute_complete` as the way to accomplish this.